src/controllers/albumController.js

```diff
@@ -7,11 +7,35 @@ const createAlbum = async (req, res) => {
   req.body.user = req.user.userId;
   const album = await Album.create(req.body);
   res.status(StatusCodes.CREATED).json({ album });
+  /*
+     #swagger.summary = 'Create new album and save it to the database'
+            #swagger.parameters['newAlbum'] = {
```

Comment:
I found that in this case we might need to add `#swagger.autoBody = false` to make sure swagger-autogen doesn't try to generate the parameter `"user"` based on finding `req.user` in the code.

```suggestion
            #swagger.autoBody = false
            #swagger.parameters['newAlbum'] = {
```